### PR TITLE
this commit is to test pr approvals, sould be deletd afterwards

### PR DESCRIPTION
**Pull request by HajjiAmine**

### CODE REVIEW

File: user-service/src/main/java/tn/esprit/UserServiceApplication.java [hunk 1]
```diff
@@ -10,5 +10,6 @@
 public class UserServiceApplication {
     public static void main(String[] args) {
         SpringApplication.run(UserServiceApplication.class, args);
+	system.output.println("this is a test");
     }
```
Comment (Mootaz-hub) [inline user-service/src/main/java/tn/esprit/UserServiceApplication.java:14]: hedha code manesh metfehmiin alih!
alesh tbedel f repo hedha 
alesh tmess fl gateway

Comment (HajjiAmine) [reply user-service/src/main/java/tn/esprit/UserServiceApplication.java:14]: glot fl ficher

Comment (Mootaz-hub) [reply user-service/src/main/java/tn/esprit/UserServiceApplication.java:13]: mela fas5ou l  pull request pr

**Review by Mootaz-hub — APPROVED** [review]
looks good to me

**Comment by Mootaz-hub** [comment]
this is draft pr to demo pr approvals

**Pull request closed without merging**